tests/integration/test_utils/data_source_config/spark_filesystem_csv.py

```diff
@@ -55,11 +67,40 @@ def __init__(
         super().__init__(config=config, data=data)
         self._base_dir = base_dir
 
+    @property
+    def _spark_session(self) -> "pyspark.SparkSession":
+        return SparkDFExecutionEngine.get_or_create_spark_session()
+
+    @property
+    def _spark_schema(self) -> Union["pyspark_types.StructType", None]:
+        from great_expectations.compatibility.pyspark import types as pyspark_types
+
+        column_types = self.config.column_types or {}
+        struct_fields = [
+            pyspark_types.StructField(column_name, column_type())
+            for column_name, column_type in column_types.items()
+        ]
+        return pyspark_types.StructType(struct_fields) if struct_fields else None
```

Comment:
Again I only ended up adding this for debugging, but I think it's valuable. Now you can pass `column_types` to Spark tests and it will convert that to a spark schema.